src/web/lyd_og_bilde/lyd_og_bilde.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Lyd og bilde på nettsider
+title: "HTML: Ekstra! Legg til lyd og video på nettsiden din"
```

Comment:
Entusiatisk! 👍 Men tittelen blir nokså lang for indexen. Kanskje kan entusiasmen flyttes til introduksjonen (som vises ved `:hover`)?

![skjermbilde 2016-07-05 kl 11 38 17](https://cloud.githubusercontent.com/assets/4810521/16580501/1aa43e92-42a5-11e6-807e-95abb495c890.png)